packages/common/rollup-configs.js

```diff
@@ -1,11 +1,13 @@
 import { execSync } from 'child_process';
-/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable import/no-extraneous-dependencies -- avoid pulling Rollup dependencies into common package.json */
 import nodeResolve from '@rollup/plugin-node-resolve';
 import typescript from '@rollup/plugin-typescript';
 import analyzer from 'rollup-plugin-analyzer';
 import dts from 'rollup-plugin-dts';
 /* eslint-enable import/no-extraneous-dependencies */
 
+const rootDir = process.env.PROJECT_CWD;
```

Comment:
This env. variable is automatically set by Yarn: https://yarnpkg.com/advanced/lifecycle-scripts/#environment-variables